src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    get info() {
+      // eslint doesn't allow me to run test with templates string syntax,
+      // that why I've used concatination
+      let result = 'name: ' + this.name;
+
+      result += ', chip version: ' + this.version;
+      result += ', wheels: ' + this.wheels;
+
+      return result;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(value = 1) {
+      if (value < 0) {
+        return this;
+      }
+
+      this.coords.y += value;
+
+      return this;
+    },
```

Comment:
If you rewrite like this it will be better
```suggestion
    goForward(step = 1) {
      if (step > 0) {
        this.coords.y += step;
      }

      return this;
    },
```